src/main/Config.cpp

```diff
@@ -151,7 +151,7 @@ Config::Config() : NODE_SEED(SecretKey::random())
     LEDGER_PROTOCOL_VERSION = CURRENT_LEDGER_PROTOCOL_VERSION;
     LEDGER_PROTOCOL_MIN_VERSION_INTERNAL_ERROR_REPORT = 18;
 
-    OVERLAY_PROTOCOL_MIN_VERSION = 32;
+    OVERLAY_PROTOCOL_MIN_VERSION = 33;
     OVERLAY_PROTOCOL_VERSION = 34;
```

Comment:
We should bump current overlay version as well

Reply:
Done